app/src/main/java/com/evercheck/flagly/developeroptions/FeatureFlagHandlerPresenter.kt

```diff
@@ -28,16 +31,24 @@ class FeatureFlagHandlerPresenter @Inject constructor(
 
     override fun onViewReady() {
         view?.setup()
-        setupFeatureFlagValues()
+    }
+
+    override fun onLoadFeatureFlagValues(search: String) {
+        this.search = search
+        this.setupFeatureFlagValues()
     }
 
     private fun setupFeatureFlagValues() {
         launch {
-            val values = withContext(coroutineContextProvider.backgroundDispatcher) {
-                featureFlagProvider.provideAppSupportedFeatureflags().map { featureFlag ->
-                    getFeatureFlagValue(featureFlag)
+            val values =
+                withContext(coroutineContextProvider.backgroundDispatcher) {
```

Comment:
remove the withContext and the launch, there is no suspend fun in this scope

Reply:
Done